VogtPizza/PizzaManager.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPizza
+{
+    class PizzaManager
+    {
+        private Pizza _pizza;
+        public PizzaManager(Pizza pizza)
+        {
+            _pizza = pizza;
+        }
+
+        public Pizza SwitchOnChoiceForIngredients(int choice)
```

Comment:
I would call this `AddIngredients`

Then you'd have a PizzaBuilder with an AddIngredients function. If you DO want to be specific about the fact that this takes a choice, I'd recommend having `AddIngredientsByChoice`. Why? Because what if you have other ways to add ingredients. `AddIngredientsByName` `AddIngredientsByChoice` etc.. you'd have the same prefix `AddIngredients` and simply change the suffix based on how you're adding.

**Keep the naming consistent/organized so its easier for the next developer if you get hit by a bus and arent around to explain the code**